modules/core/src/main/scala/gem/config/InstrumentConfig.scala

```diff
@@ -13,6 +13,15 @@ sealed abstract class InstrumentConfig extends Product with Serializable {
       case f2: F2Config     => Some(f2.key)
       case GenericConfig(_) => None
     }
+
+  def instrument: Instrument = {
+    import gem.enum.Instrument._
+
+    this match {
```

Comment:
Does this mean that we get the F2 config if we have read an Instrument form the db?